identifiers/country-us/census_autogenerated_14/us_sldu.csv

```diff
@@ -1766,7 +1766,7 @@ ocd-division/country:us/state:ut/sldu:29,sldu-49029,
 ocd-division/country:us/state:vt/sldu:addison,sldu-50ADD,
 ocd-division/country:us/state:vt/sldu:bennington,sldu-50BEN,
 ocd-division/country:us/state:vt/sldu:caledonia,sldu-50CAL,
-ocd-division/country:us/state:vt/sldu:grand_isle-chittenden,sldu-50CGI,

```

Comment:
I see the change of `ocd-division/country:us/state:vt/sldu:grand_isle-chittenden` to `ocd-division/country:us/state:vt/sldu:grand_isle` is also in #349 . I think that should be left out of this PR.